google-cloud-pubsub/.toys.rb

```diff
@@ -24,3 +24,23 @@
            path: "toys/gapic",
            update: true
 end
+
+tool "samples" do
+  tool "gen-protos" do
+    include :exec, e: true
+    include :gems
+
+    def run
+      gem "grpc-tools", "~> 1.65"
```

Comment:
Start the run with `Dir.chdir context_directory`. This ensures that you start in the base `google-cloud-pubsub` directory. (Otherwise if you're in a subdirectory like the `samples` directory when you run this tool, the next chdir below will fail because there isn't a `samples/samples/acceptance`.)